Add missing step on "setting up google tag manager" doc (#1363)
vtex-apps/io-documentation: docs/en/Recipes/store-management/setting-up-google-tag-manager.md

> 4. Click on **More Settings**, **Fields to Set** and click on `Add Field`.
> 5. In **Field**, type `location` and in **Value** `{{Original Location}}`.
> 6. Click again on `Add Field` and In **Field**, type `referrer` and in **Value** `{{Original Referrer}}`.
> 7. Click again on `Add Field` and In **Field**, type `page` and in **Value** `{{Page Path}}`.

Reviewer: ```suggestion
4. Go to **More Settings > Fields to Set**.
5. Click the **Add Field** button. Then, type `location` in **Field Name** and `{{Original Location}}` in the **Value** field.
6. Click **Add Field**. Then, type `referrer` in **Field Name** and `{{Original Referrer}}` in the **Value** field.
7. Click **Add Field**. Then, type `page` in **Field Name** and `{{Page Path}}` in the **Value** field.
```

> 4. Click on **More Settings**, **Fields to Set** and click on `Add Field`.
> 5. In **Field**, type `location` and in **Value** `{{Original Location}}`.
> 6. Click again on `Add Field` and In **Field**, type `referrer` and in **Value** `{{Original Referrer}}`.

Reviewer: ```suggestion
4. Go to **More Settings > Fields to Set**.
5. Click the **Add Field** button. Then, type `location` in **Field Name** and `{{Original Location}}` in the **Value** field.
6. Click the **Add Field** button. Then, type `referrer` in **Field Name** and `{{Original Referrer}}` in the **Value** field.
```

> 7. Click again on `Add Field` and In **Field**, type `page` and in **Value** `{{Page Path}}`.
> 8. Then, go to **Ecommerce**  and tick the `Enable Enhanced Ecommerce Features`.
> 9. In `Read data from variable` select `{{ecommerceV2}}`.
> 10. Save your changes as **Google Analytics - Checkout and Order Placed**.

Reviewer: ```suggestion
7.  Click the **Add Field** button. Then, type `page` in **Field Name** and `{{Page Path}}` in the **Value** field.
8. Then, go to the **Ecommerce** section and tick the `Enable Enhanced Ecommerce Features` option.
9. Select the `{{ecommerceV2}}` option in **Read data from variable**.
10. Save your changes as **Google Analytics - Checkout and Order Placed**.
```
